apps/mobile/src/services/tx/tx-sender/execute.ts

```diff
@@ -0,0 +1,47 @@
+import { ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import type { SafeInfo } from '@/src/types/address'
+import { createConnectedWallet } from '../../web3'
```

Comment:
I know that I'm also not very consistent here, but can we use the full import with `@....`